src/runtime-rs/crates/runtimes/src/tracer.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::cmp::min;
+use std::sync::Arc;
+use std::sync::Mutex;
+
+use anyhow::Result;
+use lazy_static::lazy_static;
+use opentelemetry::global;
+use opentelemetry::runtime::Tokio;
+use tracing::{span, subscriber::NoSubscriber, Span, Subscriber};
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::Registry;
+
+const DEFAULT_JAEGER_URL: &str = "http://localhost:14268/api/traces";
```

Comment:
I think the tracer can define as a struct, and as the `struct RuntimeHanderManagerInner{}` member variable, instead of using a global variable.

Reply:
The problem is that we want the tracing start as soon as the containerd's request came in. If we make that a member field, we can only start tracing after the runtimehandler is initialized.